Makefile

```diff
@@ -23,8 +23,9 @@ TEST_BACKSTOP = AUG0104A/CR214_0300.backstop
 DATA_FILES = starcheck_data/aca_spec.json starcheck_data/ACABadPixels starcheck_data/agasc.bad \
 	starcheck_data/fid_CHARACTERIS_JUL01 starcheck_data/fid_CHARACTERIS_FEB07 \
 	starcheck_data/fid_CHARACTERISTICS starcheck_data/characteristics.yaml \
-	starcheck_data/A.tlr starcheck_data/B.tlr starcheck_data/tlr.cfg
-
+	starcheck_data/A.tlr starcheck_data/B.tlr starcheck_data/tlr.cfg \
+	starcheck_data/overlib.js starcheck_data/up.gif starcheck_data/down.gif \
+	starcheck_data/D3_LICENSE starcheck_data/d3.js starcheck_data/d3.min.js
```

Comment:
I should cut the D3 stuff as I'm planning this as a matplotlib/png PR with a follow-up sometime in the future for links to interactive plots.